pkg/kv/kvserver/queue.go

```diff
@@ -1170,17 +1170,12 @@ func (bq *baseQueue) finishProcessingReplica(
 	// Handle failures.
 	if err != nil {
 		benign := benignerror.IsBenign(err)
-		storeBenign := benignerror.IsStoreBenign(err)
 
 		// Increment failures metric.
 		//
 		// TODO(tschottdorf): once we start asserting zero failures in tests
 		// (and production), move benign failures into a dedicated category.
 		bq.failures.Inc(1)
-		if storeBenign {
-			bq.storeFailures.Inc(1)
```

Comment:
I am not sure this is incremented anywhere else so we could potentially remove it?

Reply:
oh, you're right! PTAL at new commit.